src/rs/src/text.rs

```diff
@@ -330,7 +330,7 @@ impl<'a> SkillThing<'a> {
 			Skill::envenom =>
 				Cow::from("Target equipment gains \"Give 1 poison on hit. Throttled (only triggers twice from Adrenaline)\" & \"25% chance to poison non-ranged attackers.\""),
 			Skill::epidemic =>
-				Cow::from("When any creature dies, poison opponent equal to dying creature's poison"),
+				Cow::from("When any creature dies, cardholder poisons opponent equal to dying creature's poison"),
```

Comment:
Not looking to introduce new keyword `cardholder`

Reply:
owner it is.